### DISCUSSION TITLE
Add a cask based on a python package

### DISCUSSION THREAD

**Discussion by stuaxo** [Homebrew/discussions#4658] [Writing Formulae/Casks]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Hi,
   I was wondering about adding a python package, https://github.com/rasterio/affine/issues/100 what happens about keeping it up to date - is there CI, or something like that once this has been added ?

python-affine / py-affine / py3-affine already exists as a package for linux disros.

Cheers
Stuart

**Comment by carlocab** [comment]
That looks like an open-source CLI tool, so it should be submitted as a formula to homebrew-core (instead of as a cask to homebrew-cask).

This guide might help: https://docs.brew.sh/Adding-Software-to-Homebrew
